topics/assembly/tutorials/unicycler-assembly/tutorial.md

```diff
@@ -0,0 +1,272 @@
+
+To sequence the genome we have obtained the strain from the [Yale E. coli Stock Center](http://cgsc2.biology.yale.edu/). The stock center sent us a filter paper disk infused with cells. The disk was placed in the center of an LB-agar plate. A single colony was picked and resuspended in a liquid LB medium, grown overnight, and genomic DNA was isolated. The DNA was then sequenced using two methods. To obtain high coverage, high accuracy data we used Illumina miSEQ to generated 250-bp paired end reads. To generate high length reads we used the Oxford Nanopore MinIon machine. 
+
+Our goal is to reconstruct and annotate the full genome of *E. coli* C-1. As you will see in this tutorial a combination of many short, high accuracy reads with long, error-prone reads helps us produce an almost perfect assembly.
+
+> ### Outline step-by-step
+>
+> In this tutorial, we will deal with:
+>
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Background on data and tools
+
+
+## The data
+
+In this tutorial we assemble genome using two types of input data: (1) Illumina 250 bp paired-end reads and (2) Oxford Nanopore reads. 
+
+### Illumina data
+
+We generated 9,345,897 250 bp read pairs (library preparation performed on genomic DNA fragmented to mean size of 600 bp). However, to make sure that you can complete this tutorial in a finite amount of time we have downsampled (reduced in size) this to 1,000,000 paired end reads - just enough to produce an accurate assembly.
+
+### Oxford Nanopore Data
+
+There are 12,738 [2d-reads](http://www.nature.com/nmeth/journal/v12/n4/fig_tab/nmeth.3290_SF13.html). Maximum read length is 27,518. The distribution of reads lengths looks like this:
+
+![Nanopore read length distribution](../../images/ont_length.png "Distribution of nanopore read lengths.")
+
+You can see that there many reads under the second peak with median of approximately 7.5 kb. 
+
+> ### <i class="fa fa-warning" aria-hidden="true"></i> Oxford Nanopore Data Format
+> Oxford Nanopore machines output
+ data in [fast5](http://bioinformatics.cvr.ac.uk/blog/exploring-the-fast5-format/) format that contains additional information besides sequence data. In this tutorial we assume that these data are *already* converted into [fastq](https://en.wikipedia.org/wiki/FASTQ_format). An additional tutorial dedicated to handling of fast5 datasets will be developed shortly. 
+{: .warning-box}
+
+
+## The tools
+
+In this analysis we will perform two tasks: (1) assembly and (2) annotation. Below we will briefly outline main ideas behind these two procedures and will describe the tools we will be using.
+
+### Assembly
+
+> ### <i class="fa fa-lightbulb-o" aria-hidden="true"></i> Knowing your assembly
```

Comment:
Can you replace the `<i class="fa fa-lightbulb-o" aria-hidden="true"></i>` by `{% icon comment %}`? The list of icons for the boxes can be found in `_config.yml` 
You can also add the icon for the warning box before (I will do it otherwise after merging the PR) :smile:

Reply:
I can do it also afterwards (changing the icons) 😄

Reply:
I replaced the comment icon and added the warning icon to _config.xml (the warning sign).